update-sdk-versions.sh

```
@@ -24,40 +24,40 @@ echo ""
 # Update package.json files
 echo "Updating package.json files..."
 for package in "${PACKAGES[@]}"; do
-  find . -name "package.json" -type f -not -path "*/node_modules/*" -not -path "*/private-payments/*" | while read -r file; do
+  while read -r file; do
     if grep -q "$package" "$file"; then
       # Create backup for diff
       cp "$file" "$TEMP_DIR/$(basename $file).bak"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)